shared/overview/2.x.md

```diff
@@ -98,7 +98,7 @@ The first partition, `resin-boot`, host boot important bits according to each bo
 
 ## OS Yocto Composition
 
-The OS is composed of multiple Yocto layers. The [Yocto Project](https://www.yoctoproject.org/) build system uses these layers to compile {{ $names.os.lower }} for the various [supported platforms](./supportedboards.md).
+The OS is composed of multiple Yocto layers. The [Yocto Project](https://www.yoctoproject.org/) build system uses these layers to compile {{ $names.os.lower }} for the various [supported platforms](/reference/hardware/devices/).
```

Comment:
There is a complication here. I've come across the **supported platforms** broken link in two different "sites":
* https://docs.resin.io/reference/OS/overview/2.x/#os-yocto-composition
* https://resinos.io/docs/architecture/

The link is broken in both sites, but the fix is different for each:
* docs.resin.io → `'/reference/hardware/devices/'`
* resinos.io → `'/docs/supportedboards/'`

This pull request, as it is, only fixes the problem for **docs.resin.io**. Is the right solution to do something clever with dollar variables?